### ISSUE TITLE
Define JSON data model for tasks [kb=#1892928]

### ISSUE THREAD

**Issue by schalkneethling** [mozilla/bedrock#3549]
Define basic data model for tasks data in JSON. This may or may not be used but it gives us a starting point and a more "codified", if you will, representation of the selected tasks.

## Related User Story

STORY #1 As a person who has never contributed to Mozilla before, I see options for how I could contribute to Mozilla, so I can pick a thing to do that will help Mozilla and is the best fit for me.

**Comment by schalkneethling** [comment]
#JOSN

```
[
    {
        "id": "1",
        "name": "Download Whimsy",
        "description": "Download Whimsy and open a new tab to see it in action.",
        "simple": "true",
        "categories": [
            "addons",
            "personalize"
        ],
        "url": [
            "https://addons.mozilla.org/firefox/addon/whimsycorn/"
        ]
    },
    {
        "id": "2",
        "name": "Download Mobile Firefox",
        "description": "Download Firefox for iOS and Android",
        "simple": "true",
        "categories": [
            "growth",
            "mobile"
        ],
        "url": [
            "https://www.mozilla.org/firefox/android/",
            "https://www.mozilla.org/firefox/ios/"
        ]
    },
    {
        "id": "3",
        "name": "Follow @mozparticipate",
        "description": "Follow @mozparticipate on Twitter",
        "simple": "true",
        "categories": [
            "social media"
        ],
        "url": [
            "https://twitter.com/intent/follow/?screen_name=mozparticipate"
        ]
    },
    {
        "id": "4",
        "name": "Joy of Coding",
        "description": "Watch an episode of joy of coding",
        "simple": "false",
        "categories": [
            "programming",
            "learn"
        ],
        "url": [
            "https://air.mozilla.org/the-joy-of-coding-mconley-livehacks-on-firefox-episode-1/"
        ]
    },
    {
        "id": "5",
        "name": "FoxFooding",
        "description": "Read about the foxfooding program on the Participation Hub",
        "simple": "false",
        "categories": [
            "firefoxos",
            "community"
        ],
        "url": [
            "https://firefoxos.mozilla.community/"
        ]
    },
    {
        "id": "6",
        "name": "DevTools Challenger",
        "description": "Learn more about Firefox' developer tools using the immersive DevTools Challenger",
        "simple": "false",
        "categories": [
            "programming",
            "learn",
            "tools"
        ],
        "url": [
            "http://devtoolschallenger.com/"
        ]
    },
    {
        "id": "7",
        "name": "Download Whimsy",
        "description": "Download Whimsy, sign into AMO and rate it",
        "simple": "true",
        "categories": [
            "addons",
            "personalize"
        ],
        "url": [
            "https://addons.mozilla.org/firefox/addon/whimsycorn/"
        ]
    },
    {
        "id": "8",
        "name": "Download Mobile Firefox",
        "description": "Download Firefox for iOS and Android, and sign in to Sync",
        "simple": "true",
        "categories": [
            "growth",
            "mobile"
        ],
        "url": [
            "https://www.mozilla.org/firefox/android/",
            "https://www.mozilla.org/firefox/ios/"
        ]
    },
    {
        "id": "9",
        "name": "How You Participate",
        "description": "Share your favorite way to participate with @mozparticipate on Twitter using #QA1 hashtag",
        "simple": "true",
        "categories": [
            "social media"
        ],
        "url": [
            "https://twitter.com/intent/tweet/?text=@mozparticipate&amp;hashtags=QA1"
        ]
    },
    {
        "id": "10",
        "name": "Joy of Coding",
        "description": "Watch an episode of joy of coding, log in and add a comment",
        "simple": "false",
        "categories": [
            "programming",
            "learn"
        ],
        "url": [
            "https://air.mozilla.org/the-joy-of-coding-mconley-livehacks-on-firefox-episode-1/"
        ]
    },
    {
        "id": "11",
        "name": "Join FoxFooding",
        "description": "Join the foxfooding program, sign-in to participation hub, and request a device",
        "simple": "false",
        "categories": [
            "firefoxos",
            "community"
        ],
        "url": [
            "https://firefoxos.mozilla.community/"
        ]
    },
    {
        "id": "12",
        "name": "Participate In A DevTools Challenge",
        "description": "Go to the DevTools Challenger website and do one of the challenges. For example, Editing CSS Filters",
        "simple": "false",
        "categories": [
            "programming",
            "learn",
            "tools"
        ],
        "url": [
            "http://devtoolschallenger.com/#d1600"
        ]
    },
]
```

**Comment by jpetto** [comment]
Looking great!

Might be jumping ahead a bit, but wondering if we need 2 descriptions - one for v1 (no sign-in) and one for v2 (sign-in required). Or maybe these should be arrays of steps?

**Comment by schalkneethling** [comment]
Yeah, so I was also thinking about that. Here I just added each into it's own object/section hence there are 12 as apposed to 6 but, I reckon your suggestions is good. Let me think about it and make some tweaks. Will @ you once it is ready for another round of review. Thanks!

**Comment by schalkneethling** [comment]
@jpetto Not sure if it is what you had in mind but, updated the JSON to list the steps for each task. Also, converted to a Gist so, feel free to fork and edit ;)

**Comment by alexgibson** [comment]
Is this issue still ongoing, or can it be closed along with #3611 ?
